src/shared/helpers.js

```diff
@@ -0,0 +1,129 @@
+export const regExpEmail = /^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i;
+export const regExpPassword = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/i;
+const regExpPhone = /^([+]?[\s0-9]+)?(\d{3}|[(]?[0-9]+[)])?([-]?[\s]?[0-9])+$/i;
+const regExpNumbers = /^[0-9,.]+$/i;
+const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+
+export function createFullName(items) {
+  return items.map((item) => ({
+    ...item,
+    name: `${item.name} ${item.lastName}`,
+  }));
+}
+
+export function generateId() {
+  let ID = '';
```

Comment:
We name variable in lowercase in javascript all the time. sometimes we can do this for constant variable

Reply:
ok, thanks. fixed